Content.Shared/Throwing/LandEvent.cs

```diff
@@ -9,7 +9,8 @@ namespace Content.Shared.Throwing
     /// <summary>
     /// Raised when a thrown entity is no longer moving.
     /// </summary>
-    public sealed class StopThrowEvent : EntityEventArgs
+    [ByRefEvent]
+    public record struct StopThrowEvent
     {
         public EntityUid? User;
     }
```

Comment:
```suggestion
    public record struct StopThrowEvent(EntityUid? User = null);
```
just give it a primary constructor